src/javascript/EditPanel/EditPanel.jsx

```diff
@@ -70,13 +82,15 @@ const EditPanelCmp = ({formik, title, notificationContext, client}) => {
         nodeTypeName
     };
 
-    const tabActions = registry.find({target: 'editHeaderTabsActions'});
     const SelectedTabComponents = {};
-    tabActions.forEach(action => {
-        SelectedTabComponents[action.value] = action.displayableComponent;
-    });
-
-    const [activeTab, setActiveTab] = useState('edit');
+    const tabActions = registry.find({target: 'editHeaderTabsActions'});
+    tabActions
+        .filter(tab => tab.value !== 'advanced' ||
+            (tab.value === 'advanced' && advancedTabPermission.checksResult)
```

Comment:
As tabs are now components in the registry, the permission must be set on the tab registration (in the config) and the check must be done on each tab.